src/bionics.h

```diff
@@ -142,7 +142,11 @@ struct bionic_data {
     requirement_id installation_requirement;
 
     cata::flat_set<std::string> flags;
+    cata::flat_set<std::string> active_flags;
+    cata::flat_set<std::string> inactive_flags;
```

Comment:
Consider using `flag_id`s instead of std::strings?

Reply:
Good call, I wasn't sure about those but looking closer it seems to make sense.